src/App.tsx

```diff
@@ -2,18 +2,149 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import { todos as initialTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+interface State {
+  todos: Todo[],
+  taskTitle: string,
+  selectedUser: string,
+  taskTitleError: boolean,
```

Comment:
By name, I can’t understand that it has a boolean value
```suggestion
  isTaskTitleWithError: boolean,
```